app/controllers/v1/sessions_controller.rb

```diff
@@ -175,6 +176,15 @@ def user_logout(saml_response)
       end
     end
 
+    # Diagnostic logging to determine what percentage of these issues
+    # would be resolved by an account lookup, before we implement that
+    def log_missing_uuid_info(exception)
+      return if exception&.identifier.blank?
+      accounts = Account.where(icn: exception.identifier)
```

Comment:
would attempting to match by icn OR secid increase our odds?
```
accounts = Account.where(icn: exception.attributes[:mhv_icn]).or(Account.where(sec_id: exception.attributes[:sec_id])
```

Reply:
Technically yes but I just checked production and there are 12 accounts with a secid but no ICN. Whereas there are 2,192,808 accounts with an ICN.

Reply:
I was thinking about the case where the secid would be in the SAML Response, but not the mhv_icn.